updated readme, yml files and Test_Bitbucket playbook (#22032)
demisto/content: Packs/Bitbucket/ReleaseNotes/1_0_1.md

#### Integrations
##### Bitbucket

> - Updated the Test_Bitbucket playbook so that the first task will be DeleteContext.

Reviewer: Remove this. Changes in the test playbook shouldn't be reflected

> - Updated the yaml file so that the output of the pull request commands will have information about the reviewers of the PR.

Reviewer: ```suggestion
- Updated the outputs of the pull request commands with the information on the reviewers.
```

> - Updated the README file according to the changes in the yaml file, and added a command example to the command bitbucket-pull-request-update. 

Reviewer: ```suggestion
- General documentation improvements.
```
